static/app/types/index.tsx

```diff
@@ -1449,6 +1449,7 @@ export type ReleaseWithHealth = BaseRelease &
 type ReleaseData = {
   commitCount: number;
   data: {};
+  adoptionStages?: {};
```

Comment:
should we also move this support to the end of this object since it's optional?

Reply:
Will do! :)